tests/posix/timer/prj.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_POSIX_API=y
```

Comment:
`CONFIG_POSIX_API` is likely to be deprecated soon, but we still need it for the time being to use standard include paths.

Probably would be good to add these two
```
# no AEP choice beyond the system interfaces is required
CONFIG_POSIX_AEP_CHOICE_BASE=y
# to highlight that the testsuite is for the POSIX_TIMERS Option Group
CONFIG_POSIX_TIMERS=y
```

Reply:
Thanks. Working on these.